### PR TITLE
feat: create web view component

### PR DESCRIPTION
**Pull request by vhu-axelor**
RM#75354

### CODE REVIEW

**Comment by vhu-axelor** [comment]
To test this PR you can put this on `SettingsScreen` :
```
/*
 * Axelor Business Solutions
 *
 * Copyright (C) 2024 Axelor (<http://axelor.com>).
 *
 * This program is free software: you can redistribute it and/or  modify
 * it under the terms of the GNU Affero General Public License, version 3,
 * as published by the Free Software Foundation.
 *
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU Affero General Public License for more details.
 *
 * You should have received a copy of the GNU Affero General Public License
 * along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */

import React, {useCallback} from 'react';
import {StyleSheet, View} from 'react-native';
import {
  Screen,
  SwitchCard,
  useConfig,
  useTheme,
  Text,
} from '@axelor/aos-mobile-ui';
import {useDispatch, useSelector} from 'react-redux';
import {useTranslator} from '../../i18n';
import {
  disable,
  enable,
  useEffectOnline,
  useOnline,
} from '../../features/onlineSlice';
import {ApiProviderConfig} from '../../apiProviders/config';
import {TranslationsButton} from '../components';
import {WebView} from '../../components';

const SettingsScreen = ({children}) => {
  const I18n = useTranslator();
  const Theme = useTheme();
  const online = useOnline();
  const dispatch = useDispatch();

  const {
    showFilter,
    hideVirtualKeyboard,
    toggleFilterConfig,
    toggleVirtualKeyboardConfig,
    setShowSubtitles,
    showSubtitles,
  } = useConfig();

  const {baseUrl} = useSelector(state => state.auth);

  const handleToggleConnection = useCallback(
    state => {
      if (!state) {
        dispatch(enable());
      } else {
        dispatch(disable());
      }
    },
    [dispatch],
  );

  useEffectOnline();

  const handleToggleColorBlind = useCallback(
    state => {
      if (state) {
        Theme.activateColorBlind();
      } else {
        Theme.desactivateColorBlind();
      }
    },
    [Theme],
  );

  const handleToggleSubtitles = useCallback(
    state => {
      setShowSubtitles(state);
    },
    [setShowSubtitles],
  );

  return (
    <Screen style={styles.screen}>
      <View style={styles.container}>
        <SwitchCard
          title={I18n.t('User_ShowFilter')}
          defaultValue={showFilter}
          onToggle={toggleFilterConfig}
        />
        <SwitchCard
          title={I18n.t('User_VirtualKeyboardConfig')}
          defaultValue={hideVirtualKeyboard}
          onToggle={toggleVirtualKeyboardConfig}
        />
        <SwitchCard
          title={I18n.t('User_ColorForColorBlind')}
          defaultValue={Theme.isColorBlind}
          onToggle={handleToggleColorBlind}
        />
        {ApiProviderConfig.allowConnectionBlock && (
          <SwitchCard
            title={I18n.t('User_BlockConnection')}
            defaultValue={!online.isEnabled}
            onToggle={handleToggleConnection}
          />
        )}
        <SwitchCard
          title={I18n.t('User_Show_Drawer_Subtitles')}
          defaultValue={showSubtitles}
          onToggle={handleToggleSubtitles}
        />
        {children}
        <TranslationsButton />
      </View>
      <View style={styles.footerContainer}>
        <Text>{`${I18n.t('Base_ConnectedOn')}:`}</Text>
        <Text numberOfLines={1}>{baseUrl}</Text>
      </View>
      <WebView queryParams="action.timesheet.view.my.timesheet/list/1" />
    </Screen>
  );
};

const styles = StyleSheet.create({
  screen: {
    justifyContent: 'space-between',
  },
  container: {
    marginTop: 15,
    justifyContent: 'flex-start',
    alignItems: 'center',
    flex: 1,
  },
  footerContainer: {
    justifyContent: 'center',
    alignItems: 'center',
    marginBottom: 10,
  },
});

export default SettingsScreen;
```

**Comment by gca-axelor** [comment]
need to have 2 component, 1 for aos view and 1 for other view

**Review by gca-axelor — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -57,7 +57,7 @@
     "react-native-gifted-charts": "1.3.12",
     "react-native-linear-gradient": "2.8.3",
     "react-native-mmkv": "2.4.3",
-    "react-native-pell-rich-editor": "1.8.8",
+    "react-native-pell-rich-editor": "1.9.0",
```
Comment (lme-axelor) [inline package.json:60]: revert

File: changelogs/unreleased/75354_updateLib.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "title": "package.json: update react-native-pell-rich-editor to 1.9.0 && update react-native-webview to 11.26.1",
+  "type": "feat",
+  "packages": "ui"
+}
```
Comment (lme-axelor) [inline changelogs/unreleased/75354_updateLib.json:5]: not needed

**Pull request merged**